### PR TITLE
Privacy policy

### PR DESCRIPTION
**Pull request by grantfayvor**
* Privacy policy design

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...deType":"79","endLine":12,"endColumn":31},{"ruleId":"77","severity":1,"message":"78","line":16,"column":11,"nodeType":"79","endLine":16,"endColumn":31},{"ruleId":"82","severity":1,"message":"83","line":1,"column":1,"nodeType":"84","endLine":3,"endColumn":3},{"ruleId":"82","severity":1,"message":"85","line":4,"column":1,"nodeType":"84","endLine":11,"endColumn":3},"no-native-reassign",["86"],"no-negated-in-lhs",["87"],"jsx-a11y/anchor-is-valid","The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md","JSXOpeningElement","jsx-a11y/alt-text","img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.","import/no-anonymous-default-export","Assign object to a variable before exporting as module default","ExportDefaultDeclaration","Assign arrow function to a variable before exporting as module default","no-global-assign","no-unsafe-negation"]
```
Comment (grantfayvor) [inline .eslintcache]: Can you remove the eslintcache from the commit and also add it to gitignore

File: package.json [hunk 2]
```diff
@@ -3,18 +3,23 @@
...
     "@testing-library/user-event": "^7.2.1",
+    "autoprefixer": "^9.8.6",
+    "postcss": "^7.0.35",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-icons": "^4.1.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.4"
+    "react-scripts": "^4.0.1",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.0.2"
```
Comment (grantfayvor) [inline package.json]: What are the cost-benefits for using tailwindcss considering the size of the project and change?

File: src/components/Button/BackButton.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { IoIosArrowBack } from "react-icons/io";
+import { withRouter } from 'react-router-dom';
+
+const BackButton = ({history}) => {
+  return (
+    <div
+    onClick={() => history.goBack()}
```
Comment (grantfayvor) [inline src/components/Button/BackButton.js]: Please switch to HTML buttons for accessibility

File: src/components/Button/BackButton.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+      className="regular_16 flex items-center pb-20"
+    >
+      <IoIosArrowBack style={{ marginRight: 20 }} />
```
Comment (grantfayvor) [inline src/components/Button/BackButton.js]: Use the exported icon from the figma document for consistency.

**Review by grantfayvor — COMMENTED** [review]
Please pull my changes to the branch to be in sync and also attend to the comments.

File: src/components/Button/BackButton.js (hunk 4 above)
Comment (sadeeqaji) [reply src/components/Button/BackButton.js]: Okay done

File: src/components/Button/BackButton.js (hunk 3 above)
Comment (sadeeqaji) [reply src/components/Button/BackButton.js]: Fixed

File: .eslintcache (hunk 1 above)
Comment (sadeeqaji) [reply .eslintcache]: Done

File: package.json (hunk 2 above)
Comment (sadeeqaji) [reply package.json]: There are a lot of benefits
1. It has a built-in design system
2. We won't be mixing inline styling with CSS stylesheet.
3. It is easier to implement responsive styling with Tailwind, not need to be worrying about media query.
4. Css caching

**Review by grantfayvor — COMMENTED** [review]
looks good to me.

**Pull request merged**